dags/resources/stages/parse/table_definitions/convex/ConvexFraxPool_event_LockedAdditional.json

```diff
@@ -0,0 +1,54 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "user",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "bytes32",
+                    "name": "kek_id",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "amount",
+                    "type": "uint256"
+                }
+            ],
+            "name": "LockedAdditional",
+            "type": "event"
+        },
+        "contract_address": "(SELECT DISTINCT _stakingAddress FROM ref(`FraxBooster_call_addPool`) UNION ALL SELECT '0x02577b426f223a6b4f2351315a19ecd6f357d65c' UNION ALL SELECT '0x0a08673e3d7c454e1c6b27acd059c50df6727fc9' UNION ALL SELECT '0x698137c473bc1f0ea9b85ade45caf64ef2df48d6')",
```

Comment:
There should be quotes here instead of backticks `ref('FraxBooster_call_addPool')`

Reply:
fixed. Thank you!